src/lib/Scenes/Home/Components/Sales/index.tsx

```diff
@@ -84,47 +61,34 @@ class Sales extends React.Component<Props> {
   }
 }
 
-export default createPaginationContainer(
+export default createFragmentContainer(
   Sales,
-  {
-    sales: graphql`
-      fragment Sales_sales on Sale @relay(plural: true) {
+  graphql`
+    fragment Sales_viewer on Viewer {
+      sales(live: true, is_auction: true, size: 100) {
```

Comment:
I've added `size: 100` here because ATM there's no way to get a full total without an additional request on the MP side. At some point when the number of sales increases I imagine this will be paginated away, but for now this seemed sufficient.

Reply:
👍